src/internal/m365/backup_test.go

```diff
@@ -123,15 +124,19 @@ func (suite *DataCollectionIntgSuite) TestExchangeDataCollection() {
 				ctrlOpts := control.DefaultOptions()
 				ctrlOpts.ToggleFeatures.DisableDelta = !canMakeDeltaQueries
 
+				bpc := inject.BackupProducerConfig{
+					LastBackupVersion: version.NoBackup,
```

Comment:
could you please just add a note as to why the version doesn't matter? Or set it to the current version I guess

Reply:
done